proto/p2p/p2p.proto

```diff
@@ -146,38 +153,29 @@ message ClaimedIpPort {
   bytes tx_id = 6;
 }
 
+// GetPeerList contains a bloom filter of the currently known validator IPs.
+//
+// GetPeerList must not be responded to until finishing the handshake. After the
+// handshake is completed, GetPeerlist messages should be responded to with a
+// Peerlist message containing validators that are not present in the bloom
+// filter.
+message GetPeerList {
+  BloomFilter known_peers = 1;
+}
+
 // PeerList contains network-level metadata for a set of validators.
 //
 // PeerList must be sent in response to an inbound Handshake message from a
 // remote peer a peer wants to connect to. Once a PeerList is received after
 // a Handshake message, the p2p handshake is complete and the connection is
 // established.
-
-// Peers should periodically send PeerList messages to allow peers to
-// discover each other.
```

Comment:
Isn't this comment still true?

Reply:
Going forward, peers should only need to send peerlists in response to GetPeerList messages. We currently still push gossip the IPs to gracefully handle the transition.